src/Covid19Dashboard/index.jsx

```diff
@@ -272,6 +273,33 @@ class Covid19Dashboard extends React.Component {
                   )}
               </div>
             </TabPanel>
+            <TabPanel className='gen3-dashboard_panel'>
+              <div className='gen3-dashboard_quicksight'>
+                {/* this component doesn't need the mapboxAPIToken but it's a way to make
+                sure this is the COVID19 Commons and the iframe contents will load */}
+                { mapboxAPIToken
+                  && (
+                    <div className="gen3-dashboard_container">
+                      <div className="gen3-dashboard_row">
+                        <div className="gen3-dashboard_col">
+                          <iframe className="gen3-dashboard_frame" src="https://us-east-1.quicksight.aws.amazon.com/sn/embed/share/accounts/043745044068/dashboards/6652754a-1e60-4aaf-b160-1346cbc5fc66?directory_alias=gen3-dashboards">
```

Comment:
the code lgtm but i have questions about the data we're displaying:
1. how often is it updated?
2. how is it updated? is there a cronjob? is the code public?
3. is the code pushing the data versioned? if something breaks, can we revert to an older version?

I'm asking because it looks like we are adding a new way of pushing data to the PRC. There are already multiple ways of pushing data, and as far as i know they are not documented(?)

Reply:
This URLs will be updated rarely and only if we are changing the whole dashboard, otherwise, we will update the dashboards from our end and this url will get the new changes. This URLS are dashboards from AWS Quicksight and we had a project with AWS Team in which our goal was to get amazon Quicksight dashboards into PRC and this PR is for the same.

Reply:
Is the data pulled from the https://chicagoland.pandemicresponsecommons.org `cityofchicago` project using the Gen3 API?

Reply:
yes. I am pulling the data whenever the dashboard data needs to be updated. I am doing calculation on that data and I am uploading it into the env where we have Quicksight architecture setup. You can find the demo for it in here https://cdis.slack.com/files/U030AK74946/F04F0V2CPUN/ddi_reserach_tool_demo.mp4

Reply:
I see, so you manually run the scripts and update the data. Is there a long term plan to automatize this?

Reply:
Yes, it is. But currently, the plan is to get those dashboards up and make it available to users